src/khoj/interface/web/chat.html

```diff
@@ -335,7 +348,14 @@
             box-shadow: 0 0 11px #aaa;
         }
         #chat-input {
-            font-size: medium;
+            font-family: roboto, karma, segoe ui, sans-serif;
+            font-size: 18px;
+            height: 44px;
```

Comment:
Why is a fixed height required here?

Reply:
For aesthetic purposes, this height is set so that the initial height contains only one line, like this:
![9d721f03701cdd062a5f248924614580](https://github.com/khoj-ai/khoj/assets/20761814/f6738927-1b7c-4250-97e0-0124c46a472a)
If I remove it, the textarea is too large.
![e32b33ca97e8e24bb42493d6f0473fa2](https://github.com/khoj-ai/khoj/assets/20761814/76ccf407-19dc-4bcf-bae7-d52d7ee84472)
---
So the height of textarea is related with the font-size and I fix them both.

Reply:
I see, makes sense